src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -442,4 +447,40 @@ public void parse_extraArgumentForIgnore_throwsParseException() throws ParseExce
         String input = String.format("-config %s --ignore-standalone-config true", CONFIG_FOLDER_ABSOLUTE);
         ArgsParser.parse(translateCommandline(input));
     }
+
+    @Test
+    public void parse_withTimezone_success() throws ParseException {
+        String input = String.format("-repos \"%s\" %s -timezone UTC+11", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertEquals(ZoneId.of("UTC+11"), cliArguments.getZoneId());
+
+        input = String.format("-repos \"%s\" %s -timezone UTC-1030", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertEquals(ZoneId.of("UTC-1030"), cliArguments.getZoneId());
+
+        input = String.format("-repos \"%s\" %s -timezone UTC+00", TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertEquals(ZoneId.of("UTC"), cliArguments.getZoneId());
+    }
+
+    @Test
+    public void parse_withoutTimezone_success() throws ParseException {
```

Comment:
Don't think this is necessary, there are already lots of other tests without passing the timezone option in.
Instead, maybe we can put a test for adding timezone option but empty argument instead.